src/app/effects/api-search.effects.ts

```diff
@@ -42,14 +42,18 @@ export class ApiSearchEffects {
 					.map((action: apiAction.SearchAction) => action.payload)
 					.switchMap(query => {
 						
-						let URIquery = encodeURIComponent(query.queryString);
+						let normalizedquery = query.queryString.replace(/ /g, '+');
+						let encodedquery = encodeURIComponent(query.queryString);
+						if(normalizedquery[0] === '#') {
```

Comment:
What if the query is of type `fossasia #loklak`?
Then i guess this simple comparision with first character will fail. What are your views?
Should normalization be the part of an effect?